frontend/src/app/(routes)/multisig/components/common/SignTxn.tsx

```
@@ -8,23 +8,41 @@ import { Txn } from '@/types/multisig';
+  const router = useRouter();
+
+  const txnsCount = useAppSelector((state) => state.multisig.txns.Count);
+  const getCount = (option: string) => {
+    let count = 0;
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    txnsCount &&
+      txnsCount.forEach((t: any) => {
+        if (t?.computed_status?.toLowerCase() === option.toLowerCase()) {
+          count = t?.count;
+        }
+      });
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)